### PR TITLE
[SD-4715] - Add ability to test what products a content item will match

### PR DESCRIPTION
**Pull request by akintolga**
- A new tab page added to the settings/products to let administrators to test run a story against the products and check which products will match the story.

### CODE REVIEW

**Comment by akintolga** [comment]
Depends on https://github.com/superdesk/superdesk-core/pull/465

File: scripts/superdesk-products/products.js [hunk 1]
```diff
@@ -72,6 +73,9 @@
                             self.contentFilters = $filter('sortByName')(filters);
                         });
                     },
+                    testProducts: function(diff) {
```
Comment (gbbr) [inline scripts/superdesk-products/products.js]: Should this not be called `save`?

**Comment by marwoodandrew** [comment]
:+1:

**Comment by gbbr** [comment]
:+1:

**Pull request merged**